src/content/en/tools/lighthouse/audits/_shared/query.md

```diff
@@ -0,0 +1,15 @@
+Below the audit, Lighthouse displays something like `1 element fails
```

Comment:
This fails because it's an `.md` file, and doesn't have the standard headers. In this case, it's OK, but need to verify a few things and update the test suite to avoid these failures in the future.